### PR TITLE
[BSP][STM32][469-DISCO]Add flash sdram usb drivers

### PR DESCRIPTION
**Pull request by lymzzyh**
## 拉取/合并请求描述：(PR description)

添加bsp stm32/stm32f469-st-disco 的SDRAM驱动 onchip-flash qspi-flash 以及USB的驱动，相关依赖已做好 Kconfig组合测试OK 硬件测试OK 相关utest 已通过。

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other style
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality

### CODE REVIEW

**Comment by lymzzyh** [comment]
@qgyhd1234

**Comment by lymzzyh** [comment]
sf bench test result
```bash
[D/drv.qspi] qspi init succsee!                                                                               
[SFUD] Find a Micron flash chip. Size is 16777216 bytes.                                                      
[SFUD] sf_cmd flash device is initialize success.                                                             
[D/drv.qspi] qspi init succsee!                                                                               
flash already enter qspi mode                                                                                 
16 MB sf_cmd is current selected device.                                                                      
msh />sf b                                                                                                    
msh />sf bench yes                                                                                            
Erasing the sf_cmd 16777216 bytes data, waiting...                                                            
Erase benchmark success, total time: 120.726S.                                                                
Writing the sf_cmd 16777216 bytes data, waiting...                                                            
Write benchmark success, total time: 65.537S.                                                                 
Reading the sf_cmd 16777216 bytes data, waiting...                                                            
Read benchmark success, total time: 11.556S.
```

**Comment by lymzzyh** [comment]
sdram_test
```bash
sdram_test                                                                                              
[D/drv.sdram] Writing the 16777216 bytes data, waiting....                                                    
[D/drv.sdram] Write data success, total time: 0.771S.                                                         
[D/drv.sdram] start Reading and verifying data, waiting....                                                   
[D/drv.sdram] SDRAM test success!
```

File: bsp/stm32/stm32f469-st-disco/board/ports/fal_cfg.h [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+{                                                                    \
+    &stm32_onchip_flash_16k,                                         \
+    &stm32_onchip_flash_64k,                                         \
+    &stm32_onchip_flash_128k,                                        \
+    &nor_flash0,                                                     \
+}
+
+#ifdef FAL_PART_HAS_TABLE_CFG
+/* partition table */
+#define FAL_PART_TABLE                                                                                                     \
```
Comment (qgyhd1234) [inline bsp/stm32/stm32f469-st-disco/board/ports/fal_cfg.h]: 可以考虑再精简下

**Pull request merged**